Refactor and add Llama Python library build

limintang opened the pull request:
Summary: As title. To use static llama export outside QC dir.

Differential Revision: D68937637

limintang commented:
@pytorchbot label "topic: not user facing"

cccclai reviewed (approved):
Seems reasonable to me. @winskuo-quic @shewu-quic @haowhsu-quic please take a second look

limintang committed:
Refactor and add Llama Python library build (#8107)

Summary:

As title. To use static llama export outside QC dir.

Reviewed By: cccclai

Differential Revision: D68937637

limintang committed:
Merge branch 'main' into export-D68937637

limintang committed:
Address lint issue
